example/index.js

```diff
@@ -95,7 +92,19 @@ function reinstantiateTiles() {
 	ktx2loader.setTranscoderPath( 'https://unpkg.com/three@0.153.0/examples/jsm/libs/basis/' );
 	ktx2loader.detectSupport( renderer );
 
-	tiles = new TilesRenderer( url );
+	tiles = new TilesRenderer( );
+	tiles.registerPlugin(
+		new CesiumIonAuthPlugin( {
+			apiToken: params.ionAccessToken,
+			assetId: params.ionAssetId,
+		} )
+	);
+	tiles.maxByteSize = 536870912 * 2;
+	tiles.maxSize = 10000;
+	tiles.autoDisableRendererCulling = false;
+	tiles.loadSiblings = true;
```

Comment:
Some notes:
- `loadSiblings` is no longer an option on TilesRenderer
- `maxSize` is also not an option on TilesRenderer - perhaps you intended to set this on the lru cache
- `maxByteSize` is not an option on TilesRenderer, either. You're likely trying to set maxByte***s***Size on lru cache.
- It shouldn't be necessary to change `autoDisableRendererCulling`

The reason tiles aren't loading when the error is set lower is because the lru memory cache is filling up quickly (this tile set is using 1024x1024 textures) and not allowing any more tiles to load.